ops/inventories/dev.yml

```diff
@@ -1,18 +1,24 @@
 all:
+  vars:
+    ansible_user: vagrant
+    snapshots_branch: main
+    versions_branch: main
+    snapshots_directory: snapshots
+    versions_directory: version
+    declarations_directory: declarations
+    app_directory: ota
```

Comment:
> Why do we need to add this now here? If it was not needed before, do we really need to duplicate these variables?

Reply:
As explained in other comment
I find it less confusing to have all deployment related variables at the same spot.

In the dev.yml is where you might want to make some tests when deploying on vagrant.
I believe all config variables should be listed there